### ISSUE TITLE
XCFramework doesn't include Catalyst arm64 symbols

### ISSUE THREAD

**Issue by fermoya** [stripe/stripe-ios#1640]
## Summary
XCFramework isn't built with using Xcode 12 and it's missing symbols for arm64

## Steps to reproduce
* Create a sample project
* Embed XCFrameworks attached to the last release
* Select Any Mac as target
* Compile

## iOS version
iOS 14

## Installation method
Manually

## SDK version
Last version (19.4.0 at the moment)

### How the xcframework looks:
<img width="209" alt="Screenshot 2020-09-09 at 12 07 32" src="https://user-images.githubusercontent.com/11335612/92591027-0ff53b00-f295-11ea-83a0-becb524e7c26.png">

### How it should look:
<img width="419" alt="Screenshot 2020-09-09 at 12 05 09" src="https://user-images.githubusercontent.com/11335612/92591137-3dda7f80-f295-11ea-8ab4-df9fe2ce1113.png">
*NOTE*: This is a personal project, yours should include armv7 according to your deployment target

**Comment by fermoya** [comment]
I've tried to generate the framework manually myself by:
* Downloading stripe repo (master)
* Modifying condition inside strip-ios-master/ci-scripts/export_builds.sh
```
if ! xcodebuild -version | grep -q 'Xcode 12' &> /dev/null; then
  die "Please xcode-select a copy of Xcode 12."
fi
```
* Running `Xcode-select -switch <path to Xcode 12>/Contents/Developer
* Running `export_builds.sh`

But this library depends on `libStripe3DS2` which is missing the symbols too

**Comment by davidme-stripe** [comment]
Thanks for filing! We're working on this as part of #1633, and we'll start distributing an updated .xcframework for Xcode 12 once we land that. We're targeting to finish it by the time Xcode 12 is GM. For now, you can add the `davidme/swift-pm` branch of our repo from "File -> Swift Packages -> Add Package Dependency…".

**Comment by amuresia** [comment]
Works like a charm @david-stripe, thank you!

**Comment by davidme-stripe** [comment]
This is fixed in SDK 20.0.0.

**Comment by fermoya** [comment]
Hi @davidme-stripe , thanks for your response. The [Stripe.xcframework.zip](https://github.com/stripe/stripe-ios/releases/download/20.0.0/Stripe.xcframework.zip) available for version [20.0.0](https://github.com/stripe/stripe-ios/releases/tag/20.0.0) is missing catalyst symbols:
<img width="205" alt="Screenshot 2020-09-15 at 12 57 38" src="https://user-images.githubusercontent.com/11335612/93207562-0d8e5780-f753-11ea-8564-a833e7be2108.png">
Now the problem isn't missing symbols but missing framework for Catalyst:
<img width="860" alt="Screenshot 2020-09-15 at 13 00 07" src="https://user-images.githubusercontent.com/11335612/93207824-707fee80-f753-11ea-8862-d2eff4a0883e.png">

I can't use SPM for now, this is something we have triaged but we have some blockers so I'm using Stripe by installing manually.
